virttest/network.py

```diff
@@ -0,0 +1,57 @@
+import logging
```

Comment:
Could you change the file name to virttest/utils_network.py to align the existing naming style?

Reply:
Hi @dzhengfy , I'm thinking that even the utils_net.py is very big, but the functions in the file are independent from each other, so it may be better to add these 2 functions into that file, it will be more convenient as they both are related with network. What do you think? Or I should create a new file?